setup-nextcloud-app/main.js

```diff
@@ -5,13 +5,19 @@ async function main() {
     try {
         const app       = core.getInput("app", {required: true})
         const check     = core.getInput("check-code", {required: true})
+        const force     = core.getInput("force", {required: false})
         const serverDir = '../server'
 
         await exec.exec("mkdir", ["-p",`${serverDir}/apps/${app}`])
         await exec.exec("cp", ["-R", "./", `${serverDir}/apps/${app}`])
 
         process.chdir(serverDir)
-        await exec.exec("./occ", ["app:enable", app])
+
+        if (force) {
+            await exec.exec("./occ", ["-f", "app:enable", app])
+        }else {
+            await exec.exec("./occ", ["app:enable", app])
+        }
```

Comment:
if `./occ app:enable app -f` also works I'd prefer to build the array and append `-f` to it if needed. That will save us from maintaining 2 exec blocks.

Reply:
I will try